flocker/testtools/test/test_base.py

```diff
@@ -159,7 +159,7 @@ def test_something(self):
             test.getDetails(),
             ContainsDict({
                 'twisted-log': match_text_content(MatchesRegex(
-                    r'^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}[+-]\d{4} \[-\] foo$'
+                    r'^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}[+-]\d{4} \[-\] foo'
```

Comment:
:question: Adding, `, flags=re.MULTILINE` would better preserve the intent of the test, but this is OK.